patches/xunit/0001-Patching.patch

```diff
@@ -79,24 +46,38 @@ index 87c4ea80..7fdf2a3d 100644
 -
 -  <Target Name="UpdateNuSpecProperties" BeforeTargets="GenerateNuspec" DependsOnTargets="GetBuildVersion">
 -    <PropertyGroup>
+-      <SignedPath />
+-      <SignedPath Condition=" '$(SIGN_APP_SECRET)' != '' ">signed\</SignedPath>
+-      <!-- Local builds should have a '-dev' suffix on the build number -->
+-      <PrereleaseSuffix Condition=" '$(GITHUB_ACTIONS)' != 'true' ">-dev</PrereleaseSuffix>
 -      <!-- Never put the Git hash in the package version -->
--      <PackageVersion>$(BuildVersionSimple)$(PrereleaseVersion)</PackageVersion>
+-      <PackageVersion>$(BuildVersionSimple)$(PrereleaseVersion)$(PrereleaseSuffix)</PackageVersion>
 -      <!-- Pass through values we don't know ahead of time for any hand-crafted .nuspec files -->
--      <NuspecProperties>PackageVersion=$(PackageVersion);GitCommitId=$(GitCommitId);Configuration=$(Configuration)</NuspecProperties>
+-      <NuspecProperties>
+-        Configuration=$(Configuration);
+-        GitCommitId=$(GitCommitId);
+-        PackageVersion=$(PackageVersion);
+-        SignedPath=$(SignedPath);
+-      </NuspecProperties>
 -    </PropertyGroup>
 -  </Target>
 -
  </Project>
 diff --git a/src/xunit.assert/xunit.assert.csproj b/src/xunit.assert/xunit.assert.csproj
-index 40e37a96..29a41b43 100644
+index 0fdde3a4..15c5c49c 100644
 --- a/src/xunit.assert/xunit.assert.csproj
 +++ b/src/xunit.assert/xunit.assert.csproj
-@@ -4,18 +4,7 @@
-     <DefineConstants>$(DefineConstants);XUNIT_NULLABLE</DefineConstants>
+@@ -1,22 +1,11 @@
+ <Project Sdk="Microsoft.NET.Sdk">
+ 
+   <PropertyGroup>
+-    <DefineConstants Condition=" '$(TargetFramework)' == 'netstandard1.1' ">$(DefineConstants);XUNIT_NULLABLE</DefineConstants>
++    <DefineConstants Condition=" '$(TargetFramework)' == 'netstandard2.0' ">$(DefineConstants);XUNIT_NULLABLE</DefineConstants>
+     <DefineConstants Condition=" '$(TargetFramework)' == 'net6.0' ">$(DefineConstants);XUNIT_IMMUTABLE_COLLECTIONS;XUNIT_NULLABLE;XUNIT_SPAN</DefineConstants>
      <GenerateDocumentationFile>true</GenerateDocumentationFile>
      <Nullable>enable</Nullable>
--    <TargetFramework>netstandard1.1</TargetFramework>
-+    <TargetFramework>netstandard2.0</TargetFramework>
+-    <TargetFrameworks>netstandard1.1;net6.0</TargetFrameworks>
++    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Just to double check, @dotnet/source-build-internal is targeting net6.0 fine here? I see that there's a targeting pack for it in SBRP.

Reply:
Yes that is fine.